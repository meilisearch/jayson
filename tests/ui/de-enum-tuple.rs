use jayson::DeserializeFromValue;

#[derive(DeserializeFromValue)]
#[jayson(error = jayson::Error, tag = "t")]
enum Enum {
    Variant(i32),
}

fn main() {}
