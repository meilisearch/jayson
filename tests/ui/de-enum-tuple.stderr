error: Enum variants with unnamed associated data aren't supported by the DeserializeFromValue derive macro.
 --> tests/ui/de-enum-tuple.rs:6:12
  |
6 |     Variant(i32),
  |            ^^^^^
